lib/include/aws_iot_serializer.h

```diff
@@ -303,34 +314,34 @@ typedef struct AwsIotSerializerDecodeInterface
      * @param[in] iterator Pointer to iterator
      * @return AWS_IOT_SERIALIZER_SUCCESS if successful
      */
-    AwsIotSerializerError_t ( * next )( AwsIotSerializerDecoderIterator_t iterator);
+    AwsIotSerializerError_t ( * next )( AwsIotSerializerDecoderIterator_t iterator );
 
     /**
      * @brief Function to check if the iterator reached end of container
      * @param[in] iterator Pointer to iterator for the container
      * @return AWS_IOT_SERIALIZER_SUCCESS if successful
      */
-    bool ( * isEndOfContainer )( AwsIotSerializerDecoderIterator_t iterator);
+    bool ( * isEndOfContainer )( AwsIotSerializerDecoderIterator_t iterator );
 
 
 
     /**
-       * @brief Steps out of the container by updating the decoder object to next byte position
-       * after the container.
-       * The iterator **should** point to the end of the container when calling this function.
-       *
-       * @param[in] iterator Pointer to iterator for the container.
-       * @param[in] The outer decoder object to the same container.
-       * @return AWS_IOT_SERIALIZER_SUCCESS if successful
-       */
+     * @brief Steps out of the container by updating the decoder object to next byte position
+     * after the container.
+     * The iterator **should** point to the end of the container when calling this function.
+     *
+     * @param[in] iterator Pointer to iterator for the container.
+     * @param[in] The outer decoder object to the same container.
+     * @return AWS_IOT_SERIALIZER_SUCCESS if successful
+     */
     AwsIotSerializerError_t ( * stepOut )( AwsIotSerializerDecoderIterator_t iterator,
                                            AwsIotSerializerDecoderObject_t * pDecoderObject );
 
-  /**
-   * Pretty format and print the buffer.
-   * @param[in] pDataBuffer Pointer to the buffer
-   * @param[in] dataSize Size of the buffer.
-   */
+    /**
+     * Pretty format and print the buffer.
+     * @param[in] pDataBuffer Pointer to the buffer
+     * @param[in] dataSize Size of the buffer.
+     */
     void ( * print )( uint8_t * pDataBuffer,
```

Comment:
I guess we are no longer using this function. Could we remove it ?

Reply:
yes, let me remove